### ISSUE TITLE
acoular install error

### ISSUE THREAD

**Issue by KyonP** [acoular/acoular#13]
I'm trying to install Acoular on fresh installed Ubuntu 14.04.

I've followed installation steps, but in the end stuck in following error.

```

user@user-System-Product-Name:~$ pip install acoular
Collecting acoular
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Using cached acoular-16.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-Wi1Ipw/acoular/setup.py", line 21, in <module>
        import scipy.weave
    ImportError: No module named weave
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-Wi1Ipw/acoular/
```

installation of Scipy  and Weave seems fine, they can be imported but cannot install Acoular.

**Comment by esarradj** [comment]
I presume that there is something wrong with your Python installation because it says no module named weave. Did you accidentally use a different Python than that you installed scipy into?

**Comment by rudolfbyker** [comment]
Same issue here. I think it should probably be `import weave` instead of `import scipy.weave`, since weave has been moved to a separate module for newer versions of scipy: https://github.com/scipy/weave

**Comment by esarradj** [comment]
I would recommend to use an older scipy version where weave is still included. The next version of acoular will no longer depend on weave.

**Comment by rudolfbyker** [comment]
I also have other packages which depend on newer scipy versions. Are there good reasons to stick with the old versions, other than not having to fix acoular?

Your README.md says:

> a Python 2.7 installation is needed with the **latest** Numpy, **Scipy**, Traits, Chaco, scikit-learn, pytables packages available.

(emphasis mine).

**Comment by esarradj** [comment]
I was just saying that it could be less trouble for you to use a new python environment where acoular would just run instead of going to fix it. If you need interoperability with other packages there you have of course to fix it. We will not do because we are working hard on the new version which will also run using Python3.

**Comment by 1oly** [comment]
Hi,
Would you be willing to share your current development version (e.g. in a branch)? Would be happy to help out (if I can).

**Comment by gherold** [comment]
Hi, thanks for your interest in the acoular development!
There will be a new release in the very near future which will fix most of the issues that have been addressed here. I would very much recommend to wait for this before implementing any new stuff / forking acoular. As of yet, openly sharing the development branch is not planned. However, if you have any specific ideas about how you want to get involved (implementing new software features / functionality), feel free to contact us.

**Comment by 1oly** [comment]
Good to know! Looking forward to the new release.

**Comment by esarradj** [comment]
Solved with release 17.11
